tools/util.wdl

```diff
@@ -106,23 +116,31 @@ task split_string {
     }
 
     output {
-        File output_file = "output.txt"
-        Array[String] out = read_lines("output.txt")
+        File split_strings_file = "output.txt"
+        Array[String] split_strings = read_lines("output.txt")
     }
 }
 
 task calc_gene_lengths {
     input {
         File gtf
-        String outfile = basename(gtf, ".gtf.gz") + ".genelengths.txt"
+        String outfile_name = basename(gtf, ".gtf.gz") + ".genelengths.txt"
```

Comment:
Good catch. More repeating of myself, but IMO not worth the effort to fix in this PR. We can address later. I think it will require some more thought than is worth it ATM.

Reply:
Agreed on fixing in this PR vs. not. Just wanted to flag this so we were thinking about it prior to making that change later.